public/stylesheets/CustomNavbar.css

```diff
@@ -0,0 +1,210 @@
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+  }
+
+  h1{
+    font-family: Poppins;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 60px;
+
+    width: 478px;
+    background-color: rgba(19, 119, 255, 0.6);
+    margin-bottom: 30px;
+    padding-top:60px;
+    padding-left:40px;
+    padding-bottom:60px;
+    padding-right:0px;
+    color:#FFFFFF;
+  }
+
+  .btn{
```

Comment:
@LeRoy6714 can you remove everything that was copied from the original navbar.css file and not modified? Otherwise, we have two copies of all those classes and it will be difficult to understand what is being used.

As far as I'm aware, the only things you've changed/added is this .btn style to make the rounded buttons right?